tests/unit/acquisition/test_function.py

```diff
@@ -85,14 +90,37 @@ def test_expected_improvement_builder_builds_expected_improvement(
     npt.assert_array_almost_equal(acq_fn(query_at), expected)
 
 
-def test_expected_improvement() -> None:
-    def _ei(x: tf.Tensor) -> tf.Tensor:
-        n = tfp.distributions.Normal(0, 1)
-        return - x * n.cdf(-x) + n.prob(-x)
+@random_seed()
+@pytest.mark.parametrize('best, rtol, atol', [
+    (tf.constant([50.0]), 0.01, 1e-3),
+    (BRANIN_GLOBAL_MINIMUM[None], 0.01, 1e-3),
+    (BRANIN_GLOBAL_MINIMUM[None] * 1.01, 0.01, 1e-3)
```

Comment:
if we use the same tolerances for all of them, maybe don't parametrize over them?

Reply:
see proposition above